src/util.js

```diff
@@ -15,7 +15,7 @@ export function extend(obj, props) {
  *  @param {any} [value]
  */
 export function applyRef(ref, value) {
-	if (ref!=null) {
+	if (ref!=null&&ref!==false) {
```

Comment:
couldn't this just be `if (ref)`? so it's just a check for truthiness? Otherwise you run into the same problem if it's undefined for some reason

Reply:
Updated.